ovos_messagebus/event_handler.py

```diff
@@ -34,12 +34,18 @@ def on(self, event_name, handler):
         self.emitter.on(event_name, handler)
 
     def on_message(self, message):
-        LOG.debug(message)
+
         try:
             deserialized_message = Message.deserialize(message)
         except Exception:
             return
 
+        filter_ogs = ["gui.status.request"]
```

Comment:
Consider making this configurable with a default value.  maybe `Configuration["logs"]["excluded_messages"]`?
Specific platforms might have messages that shouldn't be logged (i.e. containing sensitive data, or encoded byte data that would be useless in logs)

Reply:
opened an issue for this, that's a feature, this PR is a quick bug fix